package.json

```diff
@@ -3,7 +3,6 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "postinstall": "sed -i -- 's/class Icon extends PureComponent {/class Icon extends PureComponent {constructor(props) { super(props) }/g' node_modules/react-native-vector-icons/lib/create-icon-set.js && patch-package && sed -i '' 's/ArrayList<HashMap<String, String>>\\[\\] packages = this.reactNativeModules/ArrayList<HashMap<String, String>> packages = this.reactNativeModules/g' node_modules/@react-native-community/cli-platform-android/native_modules.gradle",
     "start": "node node_modules/react-native/local-cli/cli.js start",
```

Comment:
Is this common? I expected just `react-native start` here. Not sure it makes any difference, but can't hurt to note.

Reply:
It's not common, I don't think that any of the scripts are actually needed and over time I would probably remove them from the readme but to keep the PR small I copied everything over. The postinstall script was actually causing issues when building and was not needed so I opted to remove it.